src/gc.c

```diff
@@ -85,14 +93,17 @@ void gc_updateStats(RedisSearchCtx *sctx, GarbageCollectorCtx *gc, size_t record
   gc->stats.totalCollected += bytesCollected;
 }
 
-size_t gc_RandomTerm(RedisModuleCtx *ctx, GarbageCollectorCtx *gc) {
+enum gc_status { STATUS_OK, INVALID_SPEC };
+
```

Comment:
Style question - not typedef? (the rest of the code base doesn't have enums afaict)

Reply:
its a habit from the dmc, will change it